botocore/parsers.py

```diff
@@ -1113,8 +1113,38 @@ def _handle_string(self, shape, text):
         return text
 
 
+class S3Parser(RestXMLParser):
+    def parse(self, response, shape):
+        parsed = super().parse(response, shape)
+        if 'Expires' in parsed:
+            parsed['ExpiresString'] = (
+                parsed.get('ResponseMetadata', {})
+                .get('HTTPHeaders', {})
+                .get('expires')
+            )
```

Comment:
I'm concerned about relying on the `expires` header here as opposed to use the original unparsed `Expires` value

Reply:
To elaborate, S3 currently models "Expires" like:
```
"Expires":{
  "shape":"Expires",
  "documentation":"<p>The date and time at which the object is no longer cacheable.</p>",
  "location":"header",
  "locationName":"Expires"
},
```
If they start sending this value under a new header name ( and update `locationName` from `Expires` to something different), this implementation breaks. I'd be in favor of storing the originally pre-parsed `Expires` value and using that for `ExpiresString` as opposed hard coding an `expires` header lookup.